### PR TITLE
Do not error on missing output

### PR DESCRIPTION
**Pull request by paurosello**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by paurosello** [comment]
@corest do you think this will introduce any problem or will break your feature? We need to ignore this error for the upgrades to work but not sure there is a better way

**Review by corest — APPROVED** [review]
Should be fine. I'm wondering how did I miss this during testing upgrades. Thx for taking care!

**Comment by paurosello** [comment]
I missed them too during testing

**Review by njuettner — APPROVED** [review]

File: service/controller/resource/tccpoutputs/create.go [hunk 1]
```diff
@@ -73,10 +73,18 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
...
 		{
 			v, err := cloudFormation.GetOutputValue(outputs, APIServerPublicLoadBalancerKey)
-			if err != nil {
-				return microerror.Mask(err)
+			// migration code to dont throw error  when the old CF Stack dont yet have the new output value
+			// TODO https://github.com/giantswarm/giantswarm/issues/13851
+			// Related: https://github.com/giantswarm/giantswarm/issues/10139
+			// after migration we can remove the check for IsOutputNotFound
+			if cloudformation.IsOutputNotFound(err) {
+				r.logger.LogCtx(ctx, "level", "debug", "message", "did not found the tenant cluster's control plane APIServerPublicLoadBalancer output")
```
Comment (anvddriesch) [inline service/controller/resource/tccpoutputs/create.go:81]:
```suggestion
				r.logger.LogCtx(ctx, "level", "debug", "message", "did not find the tenant cluster's control plane APIServerPublicLoadBalancer output")
```

**Review by anvddriesch — APPROVED** [review]

**Pull request merged**